interfaces/seccomp/backend.go

```diff
@@ -162,6 +163,69 @@ func bpfBinPath(srcName string) string {
 	return filepath.Join(dirs.SnapSeccompDir, strings.TrimSuffix(srcName, ".src")+".bin")
 }
 
+func parallelCompile(compiler Compiler, profiles []string) error {
+	if len(profiles) == 0 {
+		// no profiles, nothing to do
+		return nil
+	}
+
+	profilesQueue := make(chan string, len(profiles))
+	res := make(chan error, len(profiles))
+	cpus := runtime.NumCPU()
+
+	var wg sync.WaitGroup
+	wg.Add(cpus)
+
+	// launch as many workers as we have CPUs
+	for i := 0; i < cpus; i++ {
```

Comment:
We should probably use N = cpus-1 if cpus >=2 like we do for apparmor. Also we could also then use the min(N, len(profiles)) but not essential.